### ISSUE TITLE
`vaex.open_many` should support pass-through kwargs to `vaex.open`

### ISSUE THREAD

**Issue by thisismeleaving** [vaexio/vaex#1374]
https://github.com/vaexio/vaex/blob/e6cbb0b12c53e24521a4f4fff10409a8d86c9c2b/packages/vaex-core/vaex/__init__.py#L264

**Comment by thisismeleaving** [comment]
Such that I can disable cache when calling `open_many` e.g. 
```python
vaex.open_many([lots,of,files], fs_options={"cache": False})
```

**Comment by JovanVeljanoski** [comment]
Sounds reasonable to me. 
Would you be up for making a PR on this?

**Comment by thisismeleaving** [comment]
Looks like this has been resolved as of version 4.2.0.  Thanks!
